gradle-baseline-java/src/main/resources/META-INF/gradle-plugins/com.palantir.baseline-null-away.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2022 Palantir Technologies, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+implementation-class=com.palantir.baseline.plugins.BaselineNullAway
```

Comment:
I guess we haven't switched over to the [gradle plugin-publish 1.0](https://plugins.gradle.org/docs/publish-plugin) yet on this repo, as this would need to go in the `gradlePlugin` `implementationClass`

Reply:
I think we have, but the two definition locations are merged, and likely out of sync. This is allowed due to `duplicatesStrategy 'include'` declared here: https://github.com/palantir/gradle-baseline/blob/2399039731c200e5e9f2a866cb98df19ba4a83f0/gradle-baseline-java/build.gradle#L129-L131

I've created an issue to follow-up and migrate to the gradle-plugin definition-site: #2384